api/RTCPeerConnectionIceErrorEvent.json

```
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "address": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnectioniceerrorevent-address",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnectionIceErrorEvent/address",
+          "support": {
+            "chrome": {
+              "version_added": "81"
+            },
+            "chrome_android": {
+              "version_added": "81"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "68"
+            },
+            "opera_android": {
+              "version_added": "58"
+            },
+            "safari": {
+              "version_added": null
```

Ordinarily, I'd ask to set this value, but a test conflicts with the parent feature (14 versus 14.1)

---

I can confirm with https://mdn-bcd-collector.appspot.com/tests/api/RTCPeerConnectionIceErrorEvent that `address` and `port` are in Safari 14.1, and since that's the data for the parent feature I'll go ahead and suggest:

```suggestion
              "version_added": "14.1"
```